sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs

```diff
@@ -56,6 +56,71 @@ public class BlobBaseClient
         /// </summary>
         public virtual CustomerProvidedKey? CustomerProvidedKey => _customerProvidedKey;
 
+        /// <summary>
+        /// The Storage account name corresponding to the blob client.
+        /// </summary>
+        private string _accountName;
+
+        /// <summary>
+        /// Gets the Storage account name corresponding to the blob client.
+        /// </summary>
+        public string AccountName
+        {
+            get
+            {
+                SetNameFieldsIfNull();
+                return _accountName;
+            }
+        }
+
+        /// <summary>
+        /// The container name corresponding to the blob client.
+        /// </summary>
+        private string _containerName;
+
+        /// <summary>
+        /// Gets the container name corresponding to the blob client.
+        /// </summary>
+        public string ContainerName
```

Comment:
You need to make all your new properties `virtual` so they properly support mocking.